doc-content/enterprise-only/fuse/ba-engines-karaf-con.adoc

```diff
@@ -0,0 +1,16 @@
+[id='ba-engines-karaf-con']
+= {PRODUCT} decision and process engines with {FUSE} on Apache Karaf
+
+Apache Karaf is a polymorphic standalone open-source container. It is based on the OSGi standard from the OSGi Alliance. Karaf provides support for modularisation through OSGi bundles with sophisticated class-loading support. You can deploy multiple versions of a dependency side by side in a Karaf container. You can use hot code swapping to upgrade or replace a module without shutting down the container.
+
+{PRODUCT} integration with {FUSE} on Karaf is provided through Karaf features. You can install individual components of {PRODUCT} for {FUSE} on Karaf using these features.
+
+The following features XML files facilitate {PRODUCT} and {FUSE} on Karaf integration:
+
+* `rhba-features-<FUSE-VERSION>-features.xml`
++
+This file is part of {FUSE} installed in Karaf where `<FUSE-VERSION>` is the version of Fuse. It is stored in the Karaf system repository, in the `system/org/jboss/fuse/features/rhba-features` directory. This file contains prerequisites for installing {PRODUCT} features.
```

Comment:
Suggest "This file is stored"  I know you're avoiding the repetition, but I think it's clearer, imo.